assets/templates/account/settings/settings_base.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+    <ul class="nav nav-tabs">
+        <li class="nav-item">
+            <a class="nav-link {% block tab_account %}{% endblock %}"
+               href="{% url 'settings_account' %}">
+                Account
+            </a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link {% block tab_mail %}{% endblock %}"
+               href="{% url 'account_email' %}">
+                Mailing
+            </a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link {% block tab_authorisation %}{% endblock %}"
+               href="{% url 'account_change_password' %}">
+                Authorisation
```

Comment:
Please see definition of authorization vs authentication: https://en.wikipedia.org/wiki/Authentication#Authorization. But still, 'change password' is way more simple and direct.

Reply:
Yes, yes I did. It's now authentication in naming and the tab is called password.
The reason I wanted authentication is because we can also place Q-link relevant information on this page in the future

Reply:
It still says Authorization for me.